packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -1166,6 +1174,12 @@ mixin SchedulerBinding on BindingBase {
       return true;
     }());
   }
+
+  /// Resets the relative state of EventLoop, which is used to schedule when an error occurs
```

Comment:
What do you mean by "is used to schedule when an error occurs"?

Also: Sentences should end with `.`.

Reply:
If this is only used in tests, it should be documented as such and also be marked as `@VisibleForTesting`.

Reply:
OK